### DISCUSSION TITLE
How to edit key mapping in extension options page?

### DISCUSSION THREAD

**Discussion by kabeep** [brookhong/Surfingkeys#1895] [Q&A]
`Alt+S` is doesn't work in extension options page . A message popup with " You could not toggle Surfingkeys on its own pages . "
But in the options page , edit a key map will trigger key's event when key in .
How can I change it ?

Thanks

Chrome 107.0.5304.106
Surfingkeys 0.9.74

**Comment by b0o** [comment]
If I understand you correctly, you are having trouble entering text in the textarea on the options page, so you're trying to disable Surfingkeys on the options page? 

If that's the case, you don't need to disable Surfingkeys on the options page. The text box is an ACE editor which mimics how you edit text in vim. You need to click on the text area and press `i` to switch to insert mode, and then you can type in the box. [This section](https://github.com/brookhong/Surfingkeys#vim-editor-and-emacs-editor) of the readme might help clarify.

If I'm misunderstanding your question please let me know.

**Reply by kabeep** [reply]
> If I understand you correctly, you are having trouble entering text in the textarea on the options page, so you're trying to disable Surfingkeys on the options page?
> 
> If that's the case, you don't need to disable Surfingkeys on the options page. The text box is an ACE editor which mimics how you edit text in vim. You need to click on the text area and press `i` to switch to insert mode, and then you can type in the box. [This section](https://github.com/brookhong/Surfingkeys#vim-editor-and-emacs-editor) of the readme might help clarify.
> 
> If I'm misunderstanding your question please let me know.

That's what i need, and the problem in options page seem to has been fixed now. I can press `Backspace` to delete needless.

Thanks your answer.
